doc/tutorials/model.rst

```diff
@@ -97,11 +97,13 @@ Mathematically, we can write our model in the form
 
   \hat{y}_i = \sum_{k=1}^K f_k(x_i), f_k \in \mathcal{F}
 
-where :math:`K` is the number of trees, :math:`f` is a function in the functional space :math:`\mathcal{F}`, and :math:`\mathcal{F}` is the set of all possible CARTs. The objective function to be optimized is given by
+where :math:`K` is the number of trees, :math:`f_k` is a function in the functional space :math:`\mathcal{F}`, and :math:`\mathcal{F}` is the set of all possible CARTs. The objective function to be optimized is given by
```

Comment:
I think `f_k` is unnecessary since it's referring to the general function in the space.

Reply:
Thanks @trivialfis.  My rationale was:`f` isn't in the general function space ``:math:`\mathcal{F}` `` until it has its subscript.

Since " ``:math:`f_k` `` is a function in the functional space ``:math:`\mathcal{F}` `` "
is just a translation of " `f_k \in \mathcal{F}` " from the previous line, another alternative would be to just omit it, so that the line reads:

" where ``:math:`K` `` is the number of trees, and ``:math:`\mathcal{F}` `` is the set of all possible CARTs. "

Reply:
Thank you for the explanation.